src/main/java/com/linkedin/kmf/apps/MirrorPipelineMonitor.java

```diff
@@ -52,7 +51,7 @@
 
   private final ProduceService _produceService;
   private final ConsumeService _consumeService;
-  private final List<TopicManagementService> _topicManagementServices;
+  private final TopicManagementServiceManager _topicManagementServiceManager;
   private final String _name;
 
   public MirrorPipelineMonitor(Map<String, Object> props, String name) throws Exception {
```

Comment:
Should this class be named `MutilClusterMonitor` or `MirroredPipelineMonitor`? I just noticed that `MirrorPipelineMonitor` doesn't describe itself very well.